platform/macos/src/MGLMapView.mm

```diff
@@ -1352,10 +1359,11 @@ - (MGLMapCamera *)cameraThatFitsShape:(MGLShape *)shape direction:(CLLocationDir
 }
 
 - (MGLMapCamera *)cameraForCameraOptions:(const mbgl::CameraOptions &)cameraOptions {
-    CLLocationCoordinate2D centerCoordinate = MGLLocationCoordinate2DFromLatLng(cameraOptions.center ? *cameraOptions.center : _mbglMap->getLatLng());
+    mbgl::CameraOptions mapCamera = _mbglMap->getCameraOptions();
+    CLLocationCoordinate2D centerCoordinate = MGLLocationCoordinate2DFromLatLng(cameraOptions.center ? *cameraOptions.center : *mapCamera.center);
     double zoomLevel = cameraOptions.zoom ? *cameraOptions.zoom : self.zoomLevel;
-    CLLocationDirection direction = cameraOptions.angle ? mbgl::util::wrap(*cameraOptions.angle, 0., 360.) : self.direction;
-    CGFloat pitch = cameraOptions.pitch ? *cameraOptions.pitch : _mbglMap->getPitch();
+    CLLocationDirection direction = cameraOptions.bearing ? mbgl::util::wrap(*cameraOptions.bearing, 0., 360.) : self.direction;
```

Comment:
Wrapping was removed from `-directions`; should it have been removed here too, for consistency? Or would that exacerbate #14073?

/cc @friedbunny

Reply:
Adding it back to `-directions` fixes the issue, but I’ll look at removing it entirely.